### PR TITLE
Issue #6320: resolve remove conditional for IllegalThrowsCheck

### PR DESCRIPTION
**Pull request by rnveach**
Issue #6320
http://rveach.no-ip.org/checkstyle/regression/pitest-reports/37/pitest-coding/com.puppycrawl.tools.checkstyle.checks.coding/IllegalThrowsCheck.java.html#org.pitest.mutationtest.report.html.SourceFile@4f4e6a99_149

Regression to come.

### CODE REVIEW

File: src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/IllegalThrowsCheck.java [hunk 1]
```diff
@@ -146,11 +146,9 @@ public void visitToken(DetailAST detailAST) {
         if (!isIgnorableMethod(methodDef)) {
             DetailAST token = detailAST.getFirstChild();
             while (token != null) {
-                if (token.getType() != TokenTypes.COMMA) {
```
Comment (rnveach) [inline src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/IllegalThrowsCheck.java]: `FullIdent.createFullIdent` just creates a text string with `,`. There is no way to add a comma to `illegalClassNames` naturally as all are list strings are comma separated.

**Comment by rnveach** [comment]
Regression: http://rveach.no-ip.org/checkstyle/regression/reports/273/

No differences.

**Comment by romani** [comment]
@rnveach please generate new report, as bunch of fixes are merged.

**Comment by rnveach** [comment]
See https://github.com/checkstyle/checkstyle/issues/6320#issuecomment-537909967

**Pull request merged**